server/integration/testsuite/src/test/java/org/infinispan/server/test/client/rest/RESTOffHeapClientIT.java

```diff
@@ -0,0 +1,51 @@
+package org.infinispan.server.test.client.rest;
+
+import org.infinispan.arquillian.core.InfinispanResource;
+import org.infinispan.arquillian.core.RemoteInfinispanServer;
+import org.infinispan.server.test.category.RESTSingleNode;
+import org.jboss.arquillian.junit.Arquillian;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+/**
+ * @since 9.2
+ */
+@RunWith(Arquillian.class)
+@Category({RESTSingleNode.class})
+public class RESTOffHeapClientIT extends AbstractRESTClientIT {
+
+   @InfinispanResource("container1")
+   RemoteInfinispanServer server1;
+
+   @Override
+   protected void addRestServer() {
+      rest.addServer(server1.getRESTEndpoint().getInetAddress().getHostName(), server1.getRESTEndpoint().getContextPath());
+   }
+
+   @Override
+   protected String getDefaultCache() {
+      return "offHeapCache";
+   }
+
+   @Test
+   @Ignore
+   public void testPutDataWithTimeToLive() throws Exception {
+   }
+
+   @Test
+   @Ignore
+   public void testPutDataTTLMaxIdleCombo1() throws Exception {
+   }
+
+   @Test
+   @Ignore
+   public void testPutDataTTLMaxIdleCombo2() throws Exception {
+   }
+
+   @Test
+   @Ignore
+   public void testPutDataWithMaxIdleTime() throws Exception {
```

Comment:
In the ```DefaultDataContainer```, expiration is checked when the entries are accessed, but for the ```OffHeapDataContainer``` those 4 tests don't remove the entry for some reason

Reply:
@gustavonalle @wburns Is this a bug?

Reply:
Created https://issues.jboss.org/browse/ISPN-8370 to handle this, as this PR is to solve only the CCE

Reply:
Ok, but please use something like `@Category(Unstable.class) // See ISPN-8370` instead of `@Ignore`.

Reply:
@gustavonalle Ok, if it always fails `@Ignore` is just as good :)